website/client/components/guilds/guild.vue

```diff
@@ -0,0 +1,318 @@
+        button.btn.btn-primary Join Guild
+        br
+        button.btn.float-left(:class="[isMember ? 'btn-danger' : 'btn-success']") {{ isMember ? $t('leave') : $t('join') }}
+        br
+        button.btn.btn-purple Invite to Guild
+        br
+        button.btn.btn-purple Message Guild Leader
+        br
+        button.btn.btn-purple Donate Gems
+        br
+        button.btn.btn-primary.btn-purple(b-btn, @click="updateGuild") {{ $t('updateGuild') }}
+    div
+      h3(v-once) {{ $t('description') }}
+      p {{ guild.description }}
+      p Life hacks are tricks, shortcuts, or methods that help increase productivity, efficiency, health, and so on. Generally, they get you to a better state of life. Life hacking is the process of utilizing and implementing these secrets. And, in this guild, we want to help everyone discover these improved ways of doing things.
+    div
+      h3 Guild Information
+      h4 Welcome
+      p Below are some resources that some members might find useful. Consider checking them out before posting any questions, as they just might help answer some of them! Feel free to share your life hacks in the guild chat, or ask any questions that you might have. Please peruse at your leisure, and remember: this guild is meant to help guide you in the right direction. Only you will know what works best for you.
+    div
+      h3 Challenges
+      .card
+        h4 Challenge
+        .row
+          .col-8
+            p Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla scelerisque ultrices libero.
+          .col-4
+        .row
+          .col-md-12
+            span Tag
+            span 100
+</template>
+
+<style lang="scss" scoped>
+  .sidebar {
+    background-color: #edecee;
+  }
+
+  .card {
+    margin: 2em 0;
+    padding: 1em;
+
+    h3.leader {
+      color: #4f2a93;
+    }
+
+    .text {
+      font-size: 16px;
+      line-height: 1.43;
+      color: #4e4a57;
+    }
+  }
+
+  .guild-background {
+    background-image: linear-gradient(to bottom, rgba(237, 236, 238, 0), #edecee);
+    height: 300px;
+  }
+
+  textarea {
+    height: 150px;
+    width: 100%;
+    border-radius: 2px;
+    background-color: #ffffff;
+    border: solid 1px #c3c0c7;
+    font-size: 16px;
+    font-style: italic;
+    line-height: 1.43;
+    color: #a5a1ac;
+    padding: .5em;
+  }
+
+  .icon.shield, .icon.gem {
+    width: 40px;
+    margin-right: 1em;
+  }
+
+  .icon-row {
+    width: 200px;
+    margin-top: 3em;
+    margin-right: 3em;
+
+    .number {
+      font-size: 22px;
+      font-weight: bold;
+    }
+  }
+
+  .chat-row {
+    margin-top: 2em;
+
+    .send-chat {
+      margin-top: -3.5em;
+      z-index: 10;
+      position: relative;
+      margin-right: 1em;
+    }
+  }
+
+  .hr {
+    width: 100%;
+    height: 20px;
+    border-bottom: 1px solid #e1e0e3;
+    text-align: center;
+    margin: 2em 0;
+  }
+
+  .hr-middle {
+    font-size: 16px;
+    font-weight: bold;
+    font-stretch: condensed;
```

Comment:
this should be moved to a font-family: Roboto Condensed like in typography.scss, font-stretch doesn't work + color vars